packages/stencil-store/src/observable-map.test.ts

```diff
@@ -32,22 +32,28 @@ describe('reset', () => {
     expect(state).not.toHaveProperty('hola');
   });
 
-  test('calls reset subscriptions (fn)', () => {
-    const { subscribe, reset } = createObservableMap({ hola: 'hola' });
+
+  test('calls reset subscriptions (object)', () => {
```

Comment:
Add its own test ('calls onChange'). 
Remove clarification about subscription object, as there is no longer an option for function.